### PR TITLE
Tor: bump, userlaunch

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: net-vpn/tor/additional-files/userlaunch.in [hunk 1]
```diff
@@ -0,0 +1,8 @@
+job x-vnd.@APP@ {
+	launch @MPD_BIN@
```
Comment (fbrosson) [inline net-vpn/tor/additional-files/userlaunch.in]: I guess you meant:
```diff
-	launch @MPD_BIN@
+	launch @TOR_BIN@
```

**Comment by fbrosson** [comment]
@extrowerk, did this version of Tor pass the tests?
When I created #3114 it did not.

But if you have successfully used the program then please do not hesitate to remove the "wip-no-merge" label I've just added.

**Comment by extrowerk** [comment]
Tested it with QupZilla, works and shows in geoip that i'm routed.

**Comment by extrowerk** [comment]
Test fails tho

**Comment by fbrosson** [comment]
When a userlaunch script is installed in `$dataDir/user_launch/` does the user have the possibility to disable it or start/stop it manually?
Imho we should not ship such a script if the use cannot control it.

Hmm, there is certainly a way to integrate it in the Network settings, but did not investigate.

File: net-vpn/tor/tor-0.3.4.9.recipe [hunk 2]
```diff
@@ -90,6 +91,12 @@ BUILD()
 INSTALL()
 {
 	make install
+
+	mkdir -p $dataDir/user_launch
+	sed \
+		-e "s|@APP@|tor|" \
+		-e "s|@BIN@|"$commandBinDir"/tor$commandSuffix|" \
```
Comment (alaviss) [inline net-vpn/tor/tor-0.3.4.9.recipe]:
```suggestion
		-e "s|@BIN@|"$commandBinDir"/tor|" \
```

**Comment by alaviss** [comment]
> When a userlaunch script is installed in `$dataDir/user_launch/` does the user have the possibility to disable it or start/stop it manually?
> Imho we should not ship such a script if the use cannot control it.
> 
> Hmm, there is certainly a way to integrate it in the Network settings, but did not investigate.

IIRC there's a way to override the script by placing one in `non-packaged/data/user_launch`

**Comment by Begasus** [comment]
Any word/progress on this one?

**Comment by Begasus** [comment]
Simular approach done in: https://github.com/OscarL/haikuports/commit/3d54be3357554f453cbf07c792a19e7d1951c226

**Pull request closed without merging**